admin_guide/managing_networking.adoc

```diff
@@ -85,6 +85,38 @@ can now access any pods and services in the cluster and vice versa.
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+[[admin-guide-disabling-hostname-collision]]
+== Disabling Host Name Collision Prevention For Ingress Objects
+
+In {product-title}, host name collision prevention is enabled by default. This
+means that if a route is set to a host name, it cannot be changed by another
+route or ingress object. When disabled, {product-title} uses the route creation
+timestamp to determine the oldest route for a host name, then uses that route.
+If an older route changes its host name, or if an ingress object is introduced,
+then it can hijack a host name with a newer route. 
```

Comment:
How about:

In {product-title}, host name collision prevention is enabled by default for both routes and ingress objects. This means that the host name in a route or ingress object can only be set on creation and not edited afterwards. When disabled, {product-title} will allow you to edit the host name after creation  Because {product-title} uses the route creation timestamp to determine the oldest route or ingress object for a given host name, then uses that object if an older route changes its host name, or if an ingress object is introduced, then it can hijack a host name with a newer route.

Most installations will not need to disable this, but in Kubernetes the host names in ingress objects are allowed to be edited, and if you depend upon that behavior then you can disable the safety checks.